examples/27-gitops-toolkit.yaml

```diff
@@ -0,0 +1,24 @@
+# An example of ClusterConfig object enabling gitops toolkit
+# in a cluster:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-27
+  region: eu-north-1
+
+nodeGroups:
+  - name: ng-1
+    instanceType: m5.large
+    desiredCapacity: 1
+
+git:
+  toolkit:
+    gitProvider: github             # required. options are github or gitlab
+    owner: dr-who                   # required
+    repository: my-cluster-gitops   # required
+    personal: true                  # optional. if left false, assumes 'owner' is an org
+    branch: main                    # optional
+    namespace: "flux-system"        # optional
+    path: "flux/"                   # optional
```

Comment:
This needs to be `clusters/<cluster-name>/` since this path is used to define the cluster state not only Flux.

Reply:
ah good to know, thanks 👍